packages/blocks/src/api/raw-handling/test/blockquote-normaliser.js

```diff
@@ -1,8 +1,3 @@
-/**
- * External dependencies
- */
-import { equal } from 'assert';
```

Comment:
I just removed assert in favor of Jest built-in asserters in all these tests

Reply:
Awesome, I like this 💯